manufacturing-ai-suite/industrial-edge-insights-vision/apps/pcb-anomaly-detection/configs/pipeline-server-config.json

```diff
@@ -153,6 +153,24 @@
                     }
                 },
                 "auto_start": false
+            },
+            {
+                "name": "pcb_anomaly_detection_benchmark",
```

Comment:
Do not add the pipeline into the sample. All benchmark activities in the sample should be based on the original sample business logic.